packages/grid/test/column-rendering.test.js

```diff
@@ -15,7 +15,7 @@ import { flushGrid, getCellContent, getHeaderCellContent, onceResized } from './
     }
 
     function getBodyCell(rowIndex, columnIndex) {
-      const row = grid._getVisibleRows()[rowIndex];
+      const row = grid.$.items.children[rowIndex];
```

Comment:
`grid.$.items` is not necessarily the same as what the method returned. Note that `grid.$` is also not considered public.

Reply:
Fixed.